ACM_General/events/models.py

```diff
@@ -47,6 +47,46 @@ def get_path_for_flier(instance, filename):
     )
 
 
+class Tag(models.Model):
+    """
+    Class used for adding classification tags on events.
+    """
+
+    #: The name of the tag that is displayed on the event creation form; represented
+    #: as a CharField.
+    display_name = models.CharField(
+        verbose_name=_('Event Tag'),
+        help_text=_('Different categories for the event.'),
+        max_length=256,
+    )
+
+    #: Unique id of the tag; represented as an IntegerField.
+    id = models.IntegerField(
+        verbose_name=_('ID'),
+        help_text=_('Unique ID of the tag.'),
+        primary_key=True,
+    )
+
+    #: If the tag should be diplayed on the event creation form; represented as a
+    #: BooleanField.
+    display = models.BooleanField(
```

Comment:
I don't know if I necessarily agree with the name of this being `display`. The concept behind this was that if this field is true, the user can no longer add this tag to the event nor can it be seen on events that have already used this tag. This first part of this can be solved with https://stackoverflow.com/questions/3010489/how-do-i-filter-values-in-a-django-form-using-modelform#3013509 and a custom function on this model which is something like `is_enabled()`. I believe the name should be something like `is_disabled`. What do you think?